addon/webextension/background/startBackground.js

```diff
@@ -69,6 +71,30 @@ this.startBackground = (function() {
     return true;
   });
 
+  // We delay this check (by CHECK_MIGRATION_DELAY) just to avoid piling too
+  // many things onto browser/add-on startup
+  setTimeout(() => {
+    browser.runtime.sendMessage({funcName: "getOldDeviceInfo"}).then((result) => {
+      if (result && result.type == "success" && result.value) {
+        // There is a possible migration to run, so we'll load the entire background
+        // page and continue the process
+        return loadIfNecessary();
+      }
+      if (!result) {
+        throw new Error("Got no result from getOldDeviceInfo");
+      }
+      if (result.type == "error") {
+        throw new Error(`Error from getOldDeviceInfo: ${result.name}`);
+      }
+    }).catch((error) => {
+      if (error && error.message == "Could not establish connection. Receiving end does not exist") {
+        // Just a missing bootstrap.js, ignore
+      } else {
+        console.error("Screenshots error checking for Page Shot migration:", error);
```

Comment:
No Sentry in startBackground, because we haven't loaded catcher, senderror, etc.